pkg/irreparable/irreparable_test.go

```diff
@@ -0,0 +1,186 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package irreparable
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/auth"
+	dbx "storj.io/storj/pkg/irreparable/dbx"
+)
+
+var (
+	apiKey = []byte("")
+	ctx    = auth.WithAPIKey(context.Background(), apiKey)
+)
+
+func TestPutDoesNotExist(t *testing.T) {
```

Comment:
Tests shouldn't use `dbx` directly, this creates unnecessary dependency between them. Test the behavior without using internal structures. Code will be easier to follow and it will be more robust.

Reply:
made changes to the test case similar to satellitedb. Wish I could have seen this earlier...very nicely written.